common/decisions/wc_major_pandaria_decisions.txt

```diff
@@ -0,0 +1,444 @@
+﻿create_pandaria_decision = { # Create the Empire of Pandaria
```

Comment:
why does this decision not check for the thunder king crisis being over or not started yet? shouldn't they be related???

Reply:
also, leave a line at beginning of file and check commented code for relevance, document if needed or remove if not

Reply:
> also, leave a line at beginning of file and check commented code for relevance, document if needed or remove if not

What do you mean by this? Im not quite sure what you are trying to say - Do you want me to just leave an empty line at the top of the file and remove my code comments?

Reply:
yes please >.< 

i'd rather not have commented out warcraft code in the project if it isnt there for an explicit purpose

beginning and end of line files are just a e s t h e t i c